code/modules/events/vent_clog.dm

```diff
@@ -0,0 +1,105 @@
+/datum/round_event_control/vent_clog
+	name = "Clogged Vents: Normal"
+	typepath = /datum/round_event/vent_clog
+	weight = 10
+	max_occurrences = 3
+	min_players = 10
+
+/datum/round_event/vent_clog
+	announceWhen = 1
+	startWhen = 5
+	endWhen = 35
+	var/interval = 2
+	var/list/vents = list()
+	var/random_probability = 1
+	var/reagents_amount = 50
+	var/list/safer_chems = list(/datum/reagent/water,
+								/datum/reagent/carbon,
```

Comment:
This list is formatted wrong.

1. No tab alignment. Indent up **once** and once only.
2. This should look like:

```
var/list/safer_chems = list(
    /datum/reagent/a,
    /datum/reagent/b,
    // etc
    /datum/reagent/last,
)
```

..with every reagent being its own line, indented once, trailing comma, with the parentheses at the end.